source/help/messaging/mentioning-teammates.rst

```diff
@@ -9,6 +9,9 @@ Mentioning Teammates
 
 Use @mentions to get the attention of specific team members.
 
+.. Note::
```

Comment:
I don't know, given the confusion this can cause when people expect their team members to get actively notified, I don't think this can be overstated. Also, the current format is exactly the same as used in the _Messages_ -> _Editing Messages_ section.

See also [**my feature request**](https://mattermost.uservoice.com/forums/306457-general/suggestions/33906139-prevent-users-from-posting-ineffective-mentions-w) to make users more aware of this behavior or even _make_ edits trigger notifications, since we all know [_users don't read anything_](https://www.joelonsoftware.com/2000/04/26/designing-for-people-who-have-better-things-to-do-with-their-lives/).

Reply:
In the message editing section the note formatting is okay, since the section is only about editing posts. 

For a first time user learning about the product and mentions, being blasted with a note about not receiving a notification when editing a mention, before you understand what a mention is seems counteractive.

Would you be interested in contributing the feature for a warning dialog as you are suggesting?

Reply:
You're probably right, just having a new paragraph, right at the top of the page might be enough.

I would be very interested in a contribution, but would actually prefer not implementing a warning dialog, but rather an "intelligent" mention-management. But I feel much more discussion is needed beforehand to be aware of possible pitfalls and edge cases.  
I tried to spark this discussion on the user voice page but maybe this should move here?

Thanks again for your comments and suggestion!

Reply:
Thanks @Merlin2001. Feel free to join the "Developers" channel on our nightly build server. Our devs and community contributors have discussions in there: https://pre-release.mattermost.com/core/developers

I agree it would be better to intelligently fire mentions after edits, but I believe there was a technical block preventing this and making it very complex. Feel free to bring it up in the channel and you might get some more details.